test/unit/gateways/cyber_source_test.rb

```diff
@@ -851,6 +851,28 @@ def test_adds_application_id_as_partner_solution_id
     end.respond_with(successful_capture_response)
   ensure
     CyberSourceGateway.application_id = nil
+  def test_missing_field
```

Comment:
It looks like you accidentally deleted an `end` statement that's necessary here:

```suggestion
  ensure
    CyberSourceGateway.application_id = nil
  end

  def test_missing_field
```

Reply:
Yikes

Reply:
https://github.com/activemerchant/active_merchant/blob/63233b7e6c85f785d1636a6da0a1b42eda1ea2bd/test/unit/gateways/cyber_source_test.rb#L853